spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/implementations/ErrorHandlingFilterRowsWithErrors.scala

```diff
@@ -46,4 +47,13 @@ object ErrorHandlingFilterRowsWithErrors extends ErrorHandlingCommon {
     dataFrame.filter(!coalesce(columns: _*))
   }
 
+  /**
+   * @return BooleanType of Datatype object
+   */
+  val errorColumnType: DataType = BooleanType
+
+  /**
+   * @return None since no column added during the aggregation
```

Comment:
a potential improvement 

```suggestion
   * @return None since no error-related column added during the aggregation
```